src/pages/iou/MoneyRequestCategoryPage.js

```diff
@@ -26,13 +27,20 @@ const propTypes = {
 
     /** The report currently being used */
     report: reportPropTypes,
+
+    /* Onyx Props */
+    /** Holds data related to Money Request view state, rather than the underlying Money Request data. */
+    iou: PropTypes.shape({
```

Comment:
There's a iou proptype in `../propTypes`, as well as the default prop

Reply:
Good catch!
Done - 72bd7e84f032a9c053adfba33a58e9b30ba710ed.